lxd/auth/util.go

```diff
@@ -40,6 +44,22 @@ func EntitlementsByEntityType(entityType entity.Type) ([]Entitlement, error) {
 		return nil, fmt.Errorf("Entity type %q is not valid: %w", entityType, err)
 	}
 
+	// Some entity types do not have entitlements
```

Comment:
This function is another good candidate to be reworked by https://github.com/canonical/lxd/issues/12928